includes/admin/experiments/class-scf-admin-experiment-editor-sidebar.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Editor Sidebar Experiment
+ *
+ * This experiment allows moving field group elements to the editor sidebar.
+ *
+ * @package    Secure Custom Fields
+ * @since      6.4.3
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+if ( ! class_exists( 'SCF_Admin_Experiment_Editor_Sidebar' ) ) :
+	/**
+	 * Class SCF_Admin_Experiment_Editor_Sidebar
+	 *
+	 * Implements an experiment to move field group elements to the editor sidebar
+	 * for a cleaner interface.
+	 *
+	 * @package    Secure Custom Fields
+	 * @since      6.4.3
+	 */
+	class SCF_Admin_Experiment_Editor_Sidebar extends SCF_Admin_Experiment {
+
+		/**
+		 * Initialize the experiment.
+		 *
+		 * @return void
+		 */
+		public function initialize() {
+			$this->name        = 'editor-sidebar';
+			$this->title       = __( 'Move Elements to Editor Sidebar', 'secure-custom-fields' );
+			$this->description = __( 'Moves field group elements to the editor sidebar for a cleaner interface.', 'secure-custom-fields' );
```

Comment:
This will need some better phrasing, as it's unclear what elements it refers to and in what context. It also should be less about cleaner interface but more about the fact that custom fields can be managed using the same interface as any possible plugin integrated with the editor.

Reply:
Done.